tests/Integrations/Laravel/V5_8/EloquentSandboxedTest.php

```diff
@@ -2,44 +2,7 @@
 
 namespace DDTrace\Tests\Integrations\Laravel\V5_8;
 
-use DDTrace\Tests\Common\SpanAssertion;
-use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
-
 class EloquentSandboxedTest extends EloquentTest
 {
     const IS_SANDBOX = true;
-
-    public function testDestroy()
-    {
-        $this->connection()->exec("insert into users (id, email) VALUES (1, 'test-user-deleted@email.com')");
-        $traces = $this->tracesFromWebRequest(function () {
-            $spec  = GetSpec::create('Eloquent destroy', '/eloquent/destroy');
-            $this->call($spec);
-        });
-        $this->assertOneExpectedSpan($traces, SpanAssertion::build(
-            'eloquent.destroy',
-            'Laravel',
-            'sql',
-            'App\User'
-        )->withExactTags([
-            'integration.name' => 'eloquent',
-        ]));
-    }
-
-    public function testRefresh()
-    {
-        $this->connection()->exec("insert into users (id, email) VALUES (1, 'test-user-deleted@email.com')");
-        $traces = $this->tracesFromWebRequest(function () {
-            $spec  = GetSpec::create('Eloquent delete', '/eloquent/refresh');
-            $this->call($spec);
-        });
-        $this->assertOneExpectedSpan($traces, SpanAssertion::build(
-            'eloquent.refresh',
-            'Laravel',
-            'sql',
-            'App\User'
-        )->withExactTags([
-            'integration.name' => 'eloquent',
-        ]));
-    }
```

Comment:
Where did these go?

Reply:
Ops good catch, I did restore for L57 but not for L58. Thank you for noticing that. Restored 👍